forge-gui/res/cardsfolder/a/ana_sanctuary.txt

```diff
@@ -1,11 +1,12 @@
 Name:Ana Sanctuary
 ManaCost:2 G
 Types:Enchantment
-T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | CheckSVar$ X | Execute$ TrigPump | TriggerDescription$ At the beginning of your upkeep, if you control a blue or black permanent, target creature gets +1/+1 until end of turn. If you control a blue permanent and a black permanent, that creature gets +5/+5 until end of turn instead.
-SVar:TrigPump:DB$ Pump | ValidTgts$ Creature | TgtPrompt$ Select target creature to pump | NumAtt$ 1 | NumDef$ 1 | ConditionCheckSVar$ X | ConditionSVarCompare$ EQ1 | SubAbility$ DBPump
-SVar:DBPump:DB$ Pump | Defined$ Targeted | NumAtt$ 5 | NumDef$ 5 | ConditionCheckSVar$ X | ConditionSVarCompare$ GE2
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | IsPresent$ Permanent.YouCtrl+Blue,Permanent.YouCtrl+Black | Execute$ TrigPump | TriggerDescription$ At the beginning of your upkeep, if you control a blue or black permanent, target creature gets +1/+1 until end of turn. If you control a blue permanent and a black permanent, that creature gets +5/+5 until end of turn instead.
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature | NumAtt$ P | NumDef$ P
 SVar:Y:Count$Valid Permanent.Blue+YouCtrl/LimitMax.1
 SVar:Z:Count$Valid Permanent.Black+YouCtrl/LimitMax.1
 SVar:X:SVar$Y/Plus.Z
+SVar:P:Count$Compare X EQ2.5.1
```

Comment:
Hmn, I'm pretty sure just like with Party this shouldn't give you the bigger Pump if you only have a black+blue permanent :/